### ISSUE TITLE
Missing STL for canbus cable with difrerent shape

### ISSUE THREAD

**Issue by KianSajjadi** [bigtreetech/EBB#62]
Hey, I have the ebb sb2240 with a different cable shape than the one advertised (came in the box) and the STL doesn't fit around the connection, 

Cheers

**Comment by lixibert** [comment]
Same here, the new cable doesn't fit to the custom printed part can adapter. It seems that the rubber sleeve is way to big for it.

**Comment by Phil-x** [comment]
Same here with an sb2209 kit.

**Comment by Phil-x** [comment]
``

**Comment by kieraneglin** [comment]
Any update here?

**Comment by kieraneglin** [comment]
It's not perfect, but I've uploaded an updated model [here](https://github.com/kieraneglin/Voron-Parts/tree/master/CAN%20Cable%20Holder)

**Comment by Phil-x** [comment]
EBB published a new version last week : https://github.com/bigtreetech/EBB/blob/master/EBB%20SB2240_2209%20CAN/Custom%20Printed%20Parts/Printed_Part_for_CAN_Cable_v1.1.stl.
The fitting was way better but, to me, far from ok. If ever you're interested I published a remix of this part here : https://www.thingiverse.com/thing:6105666

**Comment by kieraneglin** [comment]
@Phil-x that didn't work for me because it interferes with the USB-C plug - I'm willing to bet that model was made for the variant that doesn't include USB

**Comment by Phil-x** [comment]
Yep. That's why I did a remix. I shaved 2 mm from the bottom to allow for the USB plug.

**Comment by Phil-x** [comment]
Hi,

Step file has been added to the thingiverse https://www.thingiverse.com/thing:6105666
:-)

Phil.

**Comment by KianSajjadi** [comment]
Cheers for the solutions, yeah I noticed that the USB-C was interfering when I printed the original and the remixed one worked well

**Comment by bennierex** [comment]
Same issue here, no updated part from @bigtreetech I see...

**Comment by JKtech3D** [comment]
Found this: https://github.com/MagicPhoenix/MPX-VORON-24R2-CBT/blob/main/Printed%20Parts/Toolhead_Can_Cable_Fix.stl
